extensions/ql-vscode/src/vscode-utils/external-files.ts

```diff
@@ -0,0 +1,50 @@
+import { Uri, window } from "vscode";
+import { AppCommandManager } from "../common/commands";
+import {
+  showAndLogExceptionWithTelemetry,
+  showBinaryChoiceDialog,
+} from "../helpers";
+import { redactableError } from "../pure/errors";
+import { asError, getErrorMessage, getErrorStack } from "../pure/helpers-pure";
+
+export async function tryOpenExternalFile(
+  CommandManager: AppCommandManager,
```

Comment:
```suggestion
  commandManager: AppCommandManager,
```

Reply:
Thanks. It keeps suggesting this capitalization and I accidentally accepted it this time.